src/parsers/uniqueTokens.js

```diff
@@ -41,75 +82,81 @@ export const parseAccountUniqueTokens = data => {
-        lastSalePaymentToken: asset.last_sale
-          ? asset.last_sale.payment_token?.symbol
-          : null,
-        type: AssetTypes.nft,
-        uniqueId:
-          asset_contract.address === ENS_NFT_CONTRACT_ADDRESS
-            ? asset.name
-            : `${get(asset_contract, 'address')}_${token_id}`,
-        urlSuffixForAsset: `${get(asset_contract, 'address')}/${token_id}`,
-      })
+      }) => {
+        const { imageUrl, lowResUrl } = handleAndSignImages(
+          asset_contract.address,
+          asset.image_url,
+          asset.image_original_url,
+          asset.image_preview_url
+        );
+        return {
+          ...pick(asset, [
+            'animation_url',
+            'current_price',
+            'description',
+            'external_link',
+            'last_sale',
+            'name',
+            'permalink',
+            'sell_orders',
+            'traits',
+          ]),
+          asset_contract: pick(asset_contract, [
+            'address',
+            'name',
+            'nft_version',
+            'schema_name',
+            'symbol',
+            'total_supply',
+          ]),
+          background: background_color ? `#${background_color}` : null,
+          collection: pick(collection, [
+            'description',
+            'discord_url',
+            'external_url',
+            'featured_image_url',
+            'hidden',
+            'image_url',
+            'name',
+            'short_description',
+            'slug',
+            'twitter_username',
+            'wiki_link',
+          ]),
+          currentPrice: asset.sell_orders
+            ? `${
+                Number(asset.sell_orders[0].current_price) / 1000000000000000000
+              } ${asset.sell_orders[0].payment_token_contract.symbol}`
+            : null,
+          familyImage: collection.image_url,
+          familyName:
+            asset_contract.address === ENS_NFT_CONTRACT_ADDRESS
+              ? 'ENS'
+              : collection.name,
+          id: token_id,
+          image_url: imageUrl,
+          isSendable:
+            asset_contract.nft_version === '1.0' ||
+            asset_contract.nft_version === '3.0' ||
+            asset_contract.schema_name === 'ERC721' ||
+            asset_contract.schema_name === 'ERC1155',
+          lastPrice: parseLastSalePrice(asset.last_sale),
+          lastPriceUsd: asset.last_sale
+            ? asset.last_sale?.payment_token?.usd_price
+            : null,
+          lastSale: asset.last_sale,
+          lastSalePaymentToken: asset.last_sale
+            ? asset.last_sale.payment_token?.symbol
+            : null,
+          lowResUrl: lowResUrl,
+          type: AssetTypes.nft,
+          uniqueId:
+            asset_contract.address === ENS_NFT_CONTRACT_ADDRESS
+              ? asset.name
+              : `${get(asset_contract, 'address')}_${token_id}`,
+          urlSuffixForAsset: `${get(asset_contract, 'address')}/${token_id}`,
```

Comment:
can we get rid of `get` here since you're touching this code and optional chaining is used already?